library/worldgen/dimension/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	moduleName = "dimension"
+	version = "1.0.0"
+	library = "worldgen"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+			testmodOnly("lifecycle_events")
```

Comment:
Does the testmod really not need resource loader?

Reply:
I wasn't expecting it to work without the resource loader either, it compiles and runs fine though (and there seem to be no resource loader being used).

Reply:
Did you run the testmod for the specific module, or did you run all the testmods?

Reply:
Looks like I was wrong. After running just the testmod, it crashed without the resource loader. The implementation doesn't seem to need it though, so `testmodOnly`?

Reply:
Yes